lib/Onyx.js

```diff
@@ -840,6 +843,31 @@ function evictStorageAndRetry(error, onyxMethod, ...args) {
         .then(() => onyxMethod(...args));
 }
 
+/**
+ * Notifys subscribers and writes current value to cache
+ *
+ * @param {String} key
+ * @param {*} value
+ * @param {String} method
+ */
+// eslint-disable-next-line rulesdir/no-negated-variables
+function notifyAndWriteToCache(key, value, method) {
```

Comment:
This looks like a false positive, due to the function name starting with `not` 

Maybe it's a good time to change the name to something more generic, that doesn't explain details like cache, AFAIK this is some common logic that runs for merges and sets, so a name like `broadcastUpdate` or `propagateUpdate` might be suitable 

Perhaps it's an opportunity to change the function name into something more abstract, effectively encapsulating the internal details, such as cache manipulation. 

Given that this function is running common logic shared by `merge` and `set`, we could consider a more generic name like `broadcastUpdate` or `propagateUpdate`. These names are not only descriptive of the function's primary task (i.e., notifying the necessary components of updates), but also abstract enough to hide the internal specifics of how this task is accomplished.

Reply:
Already fixed this rule in the `eslint-config-expensify` repo [here](https://github.com/Expensify/eslint-config-expensify/pull/68), but forgot to update the package version.

I'll change this to `broadcastUpdate` anway 👍